task/01-strings-tasks.js

```diff
@@ -69,10 +66,9 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+  return value.match(/Hello, (.+?)!/)[1];
```

Comment:
Why do you need a question mark?

Reply:
I read that this question mark choose the minimum of the words to satisfy pattern